inference-engine/src/transformations/src/transformations/low_precision/normalize_l2.cpp

```diff
@@ -17,7 +17,7 @@ using namespace ngraph;
 using namespace ngraph::pass;
 using namespace ngraph::pass::low_precision;
 
-namespace {
+namespace normalize_l2 {
```

Comment:
Why cannot we use unnamed namespaces?

Reply:
In UNITY build several source files are merged into single one. So you can have unnamed namespaces with the same symbols in it. It is a UNITY build limitation. We can wrap unnamed namespaces into named if it needed.